src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
We also expose `waitUntilComplete` on a job, a few examples:

```php
// Query job
$job = $bigQuery->runQueryAsJob($query);
$results = $job->queryResults(); // blocks

// Other jobs
$job = $table->load($data);
$job->waitUntilComplete(); // blocks
```

Does this fulfill what you are looking for?